x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/data_tier_allocation/cloud_data_tier_callout.tsx

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import React, { FunctionComponent } from 'react';
+import { EuiCallOut } from '@elastic/eui';
+
+const i18nTexts = {
+  title: i18n.translate('xpack.indexLifecycleMgmt.editPolicy.cloudDataTierCallout.title', {
+    defaultMessage: 'Create a cold tier',
+  }),
+  body: i18n.translate('xpack.indexLifecycleMgmt.editPolicy.cloudDataTierCallout.body', {
+    defaultMessage: 'Edit your deployment on Cloud to create cold tier nodes.',
```

Comment:
While the user is creating a cold tier, they're not necessarily creating cold tier nodes.
I think it's more accurate to say they're adding or assigning nodes to a cold tier.

Here are my suggestions in order of preference:
- `Add nodes to the cold tier in Elastic Cloud.`
- `Edit your Elastic Cloud deployment to add nodes to the cold tier.`
- `Edit your deployment on Elastic Cloud to add cold tier nodes.`

Reply:
Is there value in describing the mechanics of "adding nodes"? Would it be easier for the user to understand if we were to describe this in a simplified way like "Edit your deployment on Elastic Cloud to set up the cold tier."

Reply:
@cjcenizal +1. I like the idea of abstracting away "adding nodes" if we're able.

`Edit your Elastic Cloud deployment to create a cold tier.`works for me. I'll amend me original comment accordingly.